models/iaf_psc_alpha_multisynapse.cpp

```diff
@@ -122,53 +122,73 @@ iaf_psc_alpha_multisynapse::Parameters_::set( const DictionaryDatum& d )
   const double delta_EL = E_L_ - ELold;
 
   if ( updateValue< double >( d, names::V_reset, V_reset_ ) )
+  {
     V_reset_ -= E_L_;
+  }
   else
+  {
     V_reset_ -= delta_EL;
-
+  }
   if ( updateValue< double >( d, names::V_th, Theta_ ) )
+  {
     Theta_ -= E_L_;
+  }
   else
+  {
     Theta_ -= delta_EL;
-
+  }
   if ( updateValue< double >( d, names::V_min, LowerBound_ ) )
+  {
     LowerBound_ -= E_L_;
+  }
   else
+  {
     LowerBound_ -= delta_EL;
-
+  }
   updateValue< double >( d, names::I_e, I_e_ );
   updateValue< double >( d, names::C_m, C_ );
   updateValue< double >( d, names::tau_m, Tau_ );
   updateValue< double >( d, names::t_ref, TauR_ );
 
   if ( C_ <= 0 )
-    throw BadProperty( "Capacitance must be > 0." );
-
+  {
+    throw BadProperty( "Capacitance must be strictly positive." );
+  }
   if ( Tau_ <= 0. )
-    throw BadProperty( "Membrane time constant must be > 0." );
-
+  {
+    throw BadProperty( "Membrane time constant must be strictly positive." );
+  }
   std::vector< double > tau_tmp;
   if ( updateValue< std::vector< double > >( d, "tau_syn", tau_tmp ) )
   {
     for ( size_t i = 0; i < tau_tmp.size(); ++i )
     {
       if ( tau_tmp.size() < tau_syn_.size() && has_connections_ == true )
```

Comment:
This check does not depend on the loop index, so it does not need to be done for every step. It should be done before (outside of) the loop.